nginx.rewriteslashes.conf

```diff
@@ -6,8 +6,8 @@ server {
     absolute_redirect off;
 
     ## Content
-    rewrite ^/software/vue.js.html$ /software/vuejs permanent;
-    rewrite ^/de/software/vue.js.html$ /de/software/vuejs permanent;
+    rewrite ^/software/vue.js.html$ /software/vuejs.html permanent;
+    rewrite ^/de/software/vue.js.html$ /de/software/vuejs.html permanent;
```

Comment:
I don't think we need this when we dont have `cleanUrls` anymore?

Reply:
but I changed `vue.js.html` to `vuejs.html` (without the dot)